deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml

```diff
@@ -205,15 +204,11 @@ spec:
           - description: Appends JVM options to each Infinispan node.
             displayName: Extra JVM Options
             path: container.jvmOptionsAppend
-          - description: Specifies the Connector authentication method, either `Credentials` or `Keystore`.
-            displayName: Connector Authentication Type
-            path: connector.authentication.type
-          - description: Specifies the name of the Connector authentication secret.
-            displayName: Connector Authentication Secret Name
-            path: connector.authentication.secretName
-          - description: Specifies the Management authentication method, either `Credentials` or `Keystore`.
-            displayName: Management Authentication Type
-            path: management.authentication.type
-          - description: Specifies the name of the Management authentication secret.
-            displayName: Management Authentication Secret Name
-            path: management.authentication.secretName
+          - description: Name of the secrect that contains auth info
+            path: security.endpointSecret
+          - description: Select how encr cers are provided [Secret,Service]
+            path: security.endpointEncryption.type
+          - description: Platform service that serves certificates (openshift.io/service-cert is supported)
+            path: security.endpointEncryption.certService
+          - description: Name of the secrect that contains certs
```

Comment:
This is a help message for the user, please use full names, so `certificates` instead of `certs`.